packages/flutter_tools/lib/src/build_system/targets/ios.dart

```diff
@@ -486,6 +487,115 @@ class _IssueLaunchRootViewControllerAccess extends Target {
   List<Source> get outputs => <Source>[];
 }
 
+/// This target verifies that the Xcode project has an LLDB Init File set within
+/// at least one scheme.
+///
+/// LLDB Init File is needed for debugging on physical iOS 26+ devices.
+class DebugIosLLDBInit extends Target {
+  const DebugIosLLDBInit();
+
+  @override
+  String get name => 'debug_ios_lldb_init';
+
+  @override
+  List<Source> get inputs => <Source>[
```

Comment:
Can we not mark the list as `const` instead of the individual elements, or is the returned list modified somewhere?